src/docker-outside-of-docker/NOTES.md

```diff
@@ -36,7 +36,7 @@ services:
 
 - The defaults value `./` is added so that the `docker-compose.yaml` file can work when it is run outside of the container
 
-### Change the workspace to `${localWorkspaceFolder}`
+### 2. Change the workspace to `${localWorkspaceFolder}`
```

Comment:
![image](https://github.com/devcontainers/features/assets/24955913/9fa4d515-d11f-41ef-8ef6-f64123e71b67)

The diff of this PR doesn't look right to me, @gauravsaini04 can you help fix this? I'd expect this PR to only show changes made by this PR.

Reply:
This usually happens when your branch is not up-to-date with the `main` branch, or when you have merged another branch into your branch. Here are a few possible reasons:

1. Your branch is behind the main branch: If the main branch has been updated since you created your branch, and you haven't pulled the latest changes into your branch, then your PR will show those changes as differences.

2. You've merged another branch into your branch: If you've merged another branch into your branch, then all the changes from that branch will appear in your PR.

3. You've accidentally committed someone else's changes: If you've accidentally committed changes that someone else made, then those changes will appear in your PR.

To avoid this, always make sure your branch is up-to-date with the `main` branch before you start making changes. You can do this by pulling the latest changes from the `main` branch into your branch. If you've already made changes and find that your branch is behind, you can rebase your branch onto the `main` branch. This will update your branch with the latest changes from the `main` branch, and move your changes on top of those.

Reply:
Thanks Samruddhi !! Have corrected the commits to contain only my commits now. Thanks once again.👍